stdlib/shelve.pyi

```diff
@@ -1,35 +1,39 @@
-import collections.abc
 from _typeshed import Self
-from typing import Any, Dict, Iterator, Optional, Tuple
+from collections.abc import Iterator, MutableMapping
+from types import TracebackType
+from typing import Optional, Type, TypeVar, Union, overload
 
-class Shelf(collections.abc.MutableMapping[Any, Any]):
+_T = TypeVar("_T")
+_VT = TypeVar("_VT")
+
+class Shelf(MutableMapping[str, _VT]):
```

Comment:
Isn't `Generic` missing as base class? (Same for the classes below.)

Reply:
We went over this recently :). You don't need Generic as a base class if you inherit from another generic class with a typevar in its type arguments.

Reply:
Considering how well my memory works, expect me to ask again next month. 😅